### ISSUE TITLE
armv8 64bit aka RPI4

### ISSUE THREAD

**Issue by cooper7692** [HaveAGitGat/Tdarr#462]
**Describe the bug**

WARNING: The requested image's platform (linux/arm) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
standard_init_linux.go:228: exec user process caused: exec format error
**To Reproduce**

    docker run -ti \
    -v /docker/tdarr/server:/app/server \
    -v /docker/tdarr/configs:/app/configs \
    -v /docker/tdarr/logs:/app/logs \
    -v /media:/media \
    -v /transcode_cache:/temp \
    -e "serverIP=0.0.0.0" \
    -e "serverPort=8266" \
    -e "webUIPort=8265" \
    --network bridge \
    -p 8265:8265 \
    -p 8266:8266 \
    -e "TZ=Europe/London" \
    -e PUID=1000 \
    -e PGID=1000 \
    --log-opt max-size=10m \
    --log-opt max-file=5 \
    haveagitgat/tdarr_arm:2.00.10

could be as simple as making sure arm v8 is listed under supported OS/Arch in the dockerfile

**Comment by wsegatto** [comment]
Duplicate of #459.

**Comment by wsegatto** [comment]
I was able to successfully use Docker in ARMv8, but I should mention that you MUST install these packages first (both for ARMv7 and ARMv8 actually):

`sudo apt install binfmt-support qemu qemu-user-static qemu-system-*`

Also, make sure you are running the latest Docker version. I believe the error you are getting was a known bug in earlier versions.

**Comment by RampedIndent** [comment]
Above solution fixed this issue for me

**Comment by HaveAGitGat** [comment]
Linux_arm64 (arm v8) packages and containers are now available in 2.00.12:

https://tdarr.io/docs/installation/windows-linux-x64-arm-and-macos

```
  "2.00.12": {
    "darwin_x64": {
      "Tdarr_Node": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/darwin_x64/Tdarr_Node.zip",
      "Tdarr_Server": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/darwin_x64/Tdarr_Server.zip"
    },
    "linux_arm": {
      "Tdarr_Node": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_arm/Tdarr_Node.zip",
      "Tdarr_Server": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_arm/Tdarr_Server.zip"
    },
    "linux_arm64": {
      "Tdarr_Node": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_arm64/Tdarr_Node.zip",
      "Tdarr_Server": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_arm64/Tdarr_Server.zip"
    },
    "linux_x64": {
      "Tdarr_Node": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_x64/Tdarr_Node.zip",
      "Tdarr_Server": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_x64/Tdarr_Server.zip",
      "Tdarr_WebUI": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/linux_x64/Tdarr_WebUI.zip"
    },
    "win32_x64": {
      "Tdarr_Node": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/win32_x64/Tdarr_Node.zip",
      "Tdarr_Server": "https://f000.backblazeb2.com/file/tdarrs/versions/2.00.12/win32_x64/Tdarr_Server.zip"
    }
  }
```
https://hub.docker.com/layers/haveagitgat/tdarr_node/latest/images/sha256-a6a21d379bd2585f39bf10f62afaf4e48fe3f13553438e009344b22463d4e3dd?context=explore

https://hub.docker.com/layers/haveagitgat/tdarr_node/latest/images/sha256-a6a21d379bd2585f39bf10f62afaf4e48fe3f13553438e009344b22463d4e3dd?context=explore
